hedera-mirror-rest/__tests__/integrationDomainOps.js

```diff
@@ -690,43 +690,49 @@ const addAssessedCustomFee = async (assessedCustomFee) => {
 };
 
 const addCustomFee = async (customFee) => {
-  let netOfTransfers = customFee.net_of_transfers;
-  if (customFee.amount_denominator && netOfTransfers == null) {
-    // set default netOfTransfers for fractional fees
-    netOfTransfers = false;
-  }
+  customFee.fixed_fees?.forEach((f) => {
+    parseCustomFeeEntityIds(f);
+  });
+  customFee.fractional_fees?.forEach((f) => {
+    parseCustomFeeEntityIds(f);
+  });
+  customFee.royalty_fees?.forEach((f) => {
+    parseCustomFeeEntityIds(f.fallback_fee);
+  });
 
+  const fixed_fees = customFee.fixed_fees ? JSONStringify(customFee.fixed_fees) : null;
+  const fractional_fees = customFee.fractional_fees ? JSONStringify(customFee.fractional_fees) : null;
+  const royalty_fees = customFee.royalty_fees ? JSONStringify(customFee.royalty_fees) : null;
+  const table = getTableName('custom_fee', customFee);
   await pool.query(
-    `insert into custom_fee (all_collectors_are_exempt,
-                             amount,
-                             amount_denominator,
-                             collector_account_id,
-                             created_timestamp,
-                             denominating_token_id,
-                             maximum_amount,
-                             minimum_amount,
-                             net_of_transfers,
-                             royalty_denominator,
-                             royalty_numerator,
-                             token_id)
-     values ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12);`,
+    `insert into ${table} (fixed_fees,
+                             fractional_fees,
+                             royalty_fees,
+                             token_id,
+                             timestamp_range)
+     values ($1, $2, $3, $4, $5);`,
     [
-      customFee.all_collectors_are_exempt || false,
-      customFee.amount || null,
-      customFee.amount_denominator || null,
-      EntityId.parse(customFee.collector_account_id, {isNullable: true}).getEncodedId(),
-      customFee.created_timestamp.toString(),
-      EntityId.parse(customFee.denominating_token_id, {isNullable: true}).getEncodedId(),
-      customFee.maximum_amount || null,
-      customFee.minimum_amount || '0',
-      netOfTransfers != null ? netOfTransfers : null,
-      customFee.royalty_denominator || null,
-      customFee.royalty_numerator || null,
+      fixed_fees,
+      fractional_fees,
+      royalty_fees,
       EntityId.parse(customFee.token_id).getEncodedId(),
+      customFee.timestamp_range,
     ]
   );
 };
 
+const parseCustomFeeEntityIds = async (fee) => {
+  fee = {
+    all_collectors_are_exempt: true,
+    collector_account_id: '0.0.300',
+    ...fee,
+  };
```

Comment:
this creates a new object instead of adding / modifying properties to / of the existing object, the end result is then the fees, e.g., `customFee.fixed_fees[0]` is not changed at all.

do this instead

```suggestion
  Object.assign(fee, {
      all_collectors_are_exempt: true,
      collector_account_id: '0.0.300',
    },
    fee,
  };
```

Reply:
Doh, I was having some problems as that method didn't quite work either. I was able to get something similar to work, it should be fixed now.